ma/maMesh.cc

```diff
@@ -132,6 +132,22 @@ void unrotateTetXi(Vector& xi, int rotation)
   xi[0] = b[1]; xi[1] = b[2]; xi[2] = b[3];
 }
 
+/* given a set of element-local coordinates computed
+   based on the original set of vertices, this function
+   takes the rotation code and gives the coordinates
+   that match the coordinates computed based on a rotated
+   set of vertices */
+void rotateTetXi(Vector& xi, int rotation)
```

Comment:
I assume there are values of `rotation` that are not valid.  Should they be checked/asserted?

Reply:
This was done like this to be consistent with an existing function called `unrotateTetXi`.

I will add the checks for both of them in the new rebased branch.